compiler/tools/fpp-to-cpp/test/alias/TAliasAc.ref.h

```diff
@@ -0,0 +1,23 @@
+// ======================================================================
+// \title  TAliasAc.h
+// \author Generated by fpp-to-cpp
+// \brief  h file for T alias
+// ======================================================================
+
+#ifndef TAliasAc_H
+#define TAliasAc_H
+
+#include "Fw/Types/BasicTypes.h"
+
+#ifdef __cplusplus
```

Comment:
Let's omit `#ifdef __cplusplus extern "C"` here and add `extern "C"` in the hpp file.